tests/foreman/virtwho/cli/test_kubevirt.py

```diff
@@ -51,7 +51,7 @@ def virtwho_config(form_data):
     return VirtWhoConfig.create(form_data)['general-information']
 
 
-@pytest.mark.skip_if_open('BZ:1735540', reason='We have not supported kubevirt hypervisor yet')
+@pytest.mark.skip_if_open('BZ:1735540')
```

Comment:
Thanks Mike, but I prefer to use the `skipif` to keep it in line with https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/virtwho/api/test_kubevirt.py#L58
```
@pytest.mark.skipif(
    condition=(is_open('BZ:1735540')),
    reason='We have not supported kubevirt hypervisor yet',
)
```

Reply:
Or we should also modify this in https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/virtwho/api/test_kubevirt.py#L58

Reply:
I would actually prefer to use skip_if_open, as BZ skipping is the specific use for this marker. I can update api/test_kubevirt to use it as well?  Going this direction also means we don't have to import the is_open method into test modules.

Reply:
@hkx303 commit updated with both uses being consistent with `skip_if_open`, thanks for being flexible!